src/templates/addons/versionControl/index.ts

```diff
@@ -1,30 +1,63 @@
+import { prompt } from 'inquirer';
+
 import { GeneralOptions } from '@/commands/generate';
 import { copy } from '@/helpers/file';
 
-const versionControlChoices = [
-  {
-    type: 'list',
-    name: 'versionControl',
-    message: 'Which version control hosting would you like to use?',
-    choices: [
-      {
-        value: 'github',
-        name: 'GitHub',
-      },
+type VersionControlOptions = {
+  versionControlEnabled: boolean;
+  versionControlService: string;
+};
+
+const getVersionControlOptions = async (): Promise<VersionControlOptions> => {
+  const versionControlPrompt = await prompt([
+    {
+      type: 'confirm',
+      name: 'versionControlEnabled',
+      message: 'Would you like to enable version control?',
```

Comment:
```suggestion
      message: 'Would you like to enable GitHub version control?',
```

(It's GitHub specific, isn't it? Or only "git"? In either case, let's precise it 👍 )

Reply:
If you choose `Yes`, it will ask you to choose between git providers (GitHub, GitLab). So this is not only specified to GitHub